README.md

```diff
@@ -84,32 +84,50 @@ See [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_gui
 
 ## Using this collection
 
-You can call modules by their Fully Qualified Collection Name (FQCN), such as `community.network.routeros_command`.
-The following example task replaces configuration changes in the existing configuration on a network device, using the FQCN:
+You can call modules by their Fully Qualified Collection Name (FQCN), such as `community.network.a10_server`.
+The following example task creates a new a10 server, using the FQCN:
 
 ```yaml
 ---
-  - name: run command on remote devices
-    community.network.routeros_command:
-      commands: /system routerboard print
-
+    - name: Create a new server
+      community.network.a10_server:
+        host: a10.mydomain.com
+        username: myadmin
+        password: mypassword
+        partition: mypartition
+        server: test
+        server_ip: 1.1.1.100
+        server_ports:
+          - port_num: 8080
+            protocol: tcp
+          - port_num: 8443
+            protocol: TCP
 ```
 
 Alternately, you can call modules by their short name if you list the `community.network` collection in the playbook's `collections`, as follows:
 
 ```yaml
 ---
-- hosts: routeros01
-  gather_facts: false
-  connection: network_cli
+- hosts: "{{desired_inventory_group}}"
```

Comment:
This module is probably used from `localhost` anyway:
```suggestion
- hosts: localhost
```

Reply:
I'm not 100% sure about this one, I found several (perhaps not applicable) examples that appeared to use actual hosts with connection: local, but I could have been misinterpreting them:

https://github.com/a10networks/ansible-examples/blob/master/Example1_Basic_LB_Graph.yaml
https://github.com/a10networks/acos27x_ansible/blob/master/vserver.yml
https://github.com/a10networks/a10-acos-axapi/blob/master/examples/single_task/slb/virtual_server/a10_slb_virtual_server_create.yaml